ouroboros-network/src/Ouroboros/Network/PeerSelection/Governor/ActivePeers.hs

```diff
@@ -207,31 +209,52 @@ jobPromoteWarmPeer PeerSelectionActions{peerStateActions = PeerStateActions {act
                    peeraddr peerconn =
     Job job handler "promoteWarmPeer"
   where
+    -- TODO: The delay should be randomly picked from an interval.
```

Comment:
Please create an issue for it, and include its number here.

Reply:
It is fixed in #3164.